From 6248b717b4861d16b80235fd8e57d64e4f636428 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 13 Apr 2017 17:39:07 -0700
Subject: [PATCH] configure: Pass linker specific options with -Wl

This helps make it pass the options to linker correctly
and we can use non-gcc compilers

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index df4fc58..02771c6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -553,7 +553,7 @@ if test -z "$OSAF_HARDEN_FLAGS"; then
 	if echo "${CFLAGS} ${CXXFLAGS}" | grep -q -- -O0; then
 		OSAF_HARDEN_FLAGS=""
 	fi
-	OSAF_HARDEN_FLAGS="${OSAF_HARDEN_FLAGS} -fstack-protector --param ssp-buffer-size=4 -fPIE -pie -zrelro -znow"
+	OSAF_HARDEN_FLAGS="${OSAF_HARDEN_FLAGS} -fstack-protector --param ssp-buffer-size=4 -fPIE -pie -Wl,-z,relro,-z,now"
 fi
 AC_SUBST(OSAF_HARDEN_FLAGS)
 
-- 
2.12.2

